src/app/Library/CrudPanel/Traits/Create.php

```diff
@@ -256,19 +256,28 @@ private function handleManyRelationItemRemoval($modelInstance, $removedEntries,
         $forceDelete = $relationDetails['force_delete'] ?? false;
         $fallbackId = $relationDetails['fallback_id'] ?? false;
 
+        // developer provided a fallback_id he knows what he's doing, just use it.
         if ($fallbackId) {
             return $removedEntries->update([$relationForeignKey => $fallbackId]);
         }
 
+        // developer set force_delete => true, so we don't care if it's nullable or not,
+        // we just follow developers will.
```

Comment:
```suggestion
        // we just follow developer's will
```

Haha excellent phrasing!! I love it!